doc/new-guidelines.md

```diff
@@ -249,6 +274,31 @@ test macros `deftest` and `is`, which are ubiquitous in the Clojure community.
   (:require [cljs-time.coerce :as time.coerce]))
 ```
 
+### Namespace aliases
+
+Prefer the following namespace aliases:
```

Comment:
Here I'm suggesting we document only the most important aliases. Do you think we have other important aliases?

In the future, I'm sure we'll be able to automate these conventions and then we'll be able to remove this section from the guidelines.